src/app/programs/programs.component.html

```diff
@@ -1,12 +1,20 @@
-<div *ngFor="let topic of topics; index as i">
-  <h3>{{topic.subject}}</h3>
-  <div class="prog-grid">
-    <div *ngFor="let program of topic.programs" class="prog-cell">
-      <h4>{{program.name}}</h4>
-      <img src={{program.src}} alt={{program.alt}} title={{program.title}}>
-      <p>{{program.description}}</p>
-      <a href={{program.link}} class="prog-btn">Check it out</a>
```

Comment:
Show the default 3 in addition to the dynamically fetched programs (fetched programs go first)